pcre/PCRE.g4

```diff
@@ -264,8 +264,8 @@ match_point_reset
  ;
 
 quoting
- : '\\' .
- | '\\' 'Q' .*? '\\' 'E'
+ : '\\' 'Q' .*? '\\' 'E'
+ | '\\' .
```

Comment:
Is it possible to replace with the following?

```suggestion
 : '\\' ('Q' .*? '\\' 'E' | .)
```

Reply:
I just tested this on my pc and it seems to work!